index.html

```diff
@@ -518,7 +534,8 @@
 					var byteVal = dataView.getUint8(i);
 					displayByte(byteVal, i);
 				}
-				jsonobj.innerHTML = JSON.stringify(decode(dataView));
+				jsonobj.innerHTML = JSON.stringify(decode(dataView), null, 4);
```

Comment:
Any reason we can't make the indentation 2 spaces instead of 4? I generally use this tool with large objects and 4 spaces is quite a bit.